test/functionalTest/cases/0000_content_related_headers/in_out_formats.test

```diff
@@ -0,0 +1,316 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)